### ISSUE TITLE
job_worked logs unavailable unless level is debug

### ISSUE THREAD

**Issue by bgentry** [chanks/que#217]
It seems that it's not possible to get Que to output logs when jobs are worked, unless the level is set to debug. Unfortunately this also makes my logs very verbose including all SQL queries.

I believe this used to be part of the default logging setup pre-1.0. Maybe these can be altered to be `:info` level logs, or offer some way to get these to see that the queue is processing even if nothing goes wrong?

Or would you recommend that this be done by extending the job class?

**Comment by bgentry** [comment]
Yeah, I could totally see the argument for this being separate from the main library in order for it to be more flexible. The only decent argument I have to keep it in Que is that you're already doing this job timing work, so why make users do it themselves? But I'll concede it's a weak point.

I'm fine with adding my own logging + instrumentation if you think that's the right path.

**Comment by chanks** [comment]
I don't know, I think if nothing changes this will be a common complaint. And it feels pretty reasonable.

Tomorrow is my open source day at work, and I'll be looking at bit at the single-worker case in the other issue you brought up. I'll also look through the logging logic and see what feels right, I suppose.

**Comment by chanks** [comment]
Sorry for the delay, work's been crazy, but I'm on vacation next week and wanted to wrap this up before then :smile:

I just added a `log_level` job method to support configuring this log level, see the logging docs for info on how to use it. It may still make sense to change the default from DEBUG to INFO, but we'll see what people think about it. Anyway, that change is in 1.0.0.beta3, which was just released.
